src/scss/00_base/variables/fonts/_fonts.scss

```diff
@@ -0,0 +1,7 @@
+// $font-leituranews: 'Leitura News Roman 3', Arial, Helvetica, sans-serif;
```

Comment:
Maybe un-comment this file and put the comment like other files of "this file is here as an example"?
Forgot what wording we used for the other files.